.github/workflows/tii-agent-protocol-splitter.yaml

```diff
@@ -0,0 +1,46 @@
+name: tii-agent-protocol-splitter
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: [ master ]
+    tags: 
```

Comment:
tags is in wrong place. It should be under push

Reply:
Done